scripts/services/map_manager.gd

```diff
@@ -66,12 +78,26 @@ func get_pages_count(page_size):
     return pages_count
 
 func map_exists(map_name):
+    if bundled_maps.has(map_name):
+        return true
     var filepath = self.MAP_PATH + map_name + self.MAP_EXTENSION
```

Comment:
get_map_path could probably be used here, but since it returns paths to bundled maps it might not behave as expected